website/docs/developer/development-guide/documentation/documentation-dev-env.md

```diff
@@ -19,13 +19,13 @@ The installation steps below goes into further details.
 
 1. Install the latest [Node Version Manager (nvm)](https://github.com/nvm-sh/nvm) application to manage Node.js and npm versions.
    ```sh
-   curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
+   curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
    ```
 
-2. Install the proper Node.js and npm versions to your machine. Currently Node v14.17.6 should be used.
+2. Install the proper Node.js and npm versions to your machine. Currently Node v16.15.1 should be used.
    ```sh
-   nvm install v16.13.1
-   nvm use v16.13.1
+   nvm install --lts
+   nvm use --lts
```

Comment:
Auto-updating makes me a bit nervous, but I understand why it is desired from a security standpoint.

Reply:
I was being a little lazy here but you make a good point. 16.15.1 is the latest LTS and there is the possibility the LTS 18 would come out and could cause issues.I did a little reading and we could talk about limiting it to the latest version within the LTS major version so v16.x.x using the `--lts=Gallium` flag. For the meantime I reverted it to use the specific version to keep everything consistent throughout.